pkg/cmd/roachprod-microbench/executor.go

```diff
@@ -103,7 +108,20 @@ func newExecutor(config executorConfig) (*executor, error) {
 		return nil, err
 	}
 
-	binariesList := []string{config.binaries}
+	runOptions := install.DefaultRunOptions().WithRetryDisabled()
+	if !config.spot {
+		// If we are not running on spot instances we'll want to retry operations,
+		// for transient errors, in case of a preemption. But since it's possible a
```

Comment:
I agree, originally I called this flag 'mig', but it also didn't feel right. Maybe something that indicates it will use a long retry mechanism, or that it runs on recoverable VMs.